point_viewer_proto_rust/src/proto.proto

```diff
@@ -75,23 +75,51 @@ message NodeId {
   int64 deprecated_index = 2;
 }
 
-message Node {
-  enum PositionEncoding {
+enum PositionEncoding {
     INVALID = 0;
     Uint8 = 1;
     Uint16 = 2;
     Float32 = 3;
     Float64 = 4;
-  }
+}
 
+
+message Node {
   PositionEncoding position_encoding = 2;
   int64 num_points = 3;
   NodeId id = 4;
 }
 
+enum AttributeType{
```

Comment:
Missing space before opening braces

Reply:
Since it's called `data_type` below, how about unifying it everywhere to `AttributeDataType`?